### PR TITLE
Remove old project bank fields.

### PR DESCRIPTION
**Pull request by gannetson**

### CODE REVIEW

File: bluebottle/projects/tests/test_api.py [hunk 1]
```diff
@@ -403,21 +379,21 @@ def test_set_bank_details(self):
...
+    #         'account_number': 'NL18ABNA0484fesewf869868',
+    #     }
+    #
+    #     response = self.client.post(self.manage_projects_url, project_data,
+    #                                 token=self.some_user_token)
+    #
+    #     self.assertEquals(response.status_code,
+    #                       status.HTTP_400_BAD_REQUEST)
+    #     self.assertEquals(json.loads(response.content)['account_number'][0],
+    #                       'Wrong IBAN length for country code NL.')
```
Comment (eodolphi) [inline bluebottle/projects/tests/test_api.py]: Shouldn't we keep this test? If not it should just be removed.

File: bluebottle/bb_projects/models.py [hunk 2]
```diff
@@ -193,24 +193,11 @@ class BaseProject(models.Model, GetTweetMixin):
         related_name="project_account_holder_country")
 
     # Bank details
-    account_iban = IBANField(_("account IBAN"), null=True, blank=True)
+    account_number = models.CharField(_("Account number"), max_length=255, null=True, blank=True)
```
Comment (eodolphi) [inline bluebottle/bb_projects/models.py]: This also requires a change in reef right?

File: bluebottle/bb_projects/serializers.py [hunk 3]
```diff
@@ -207,7 +201,5 @@ class Meta:
                   'organization', 'language', 'account_holder_name',
                   'account_holder_address', 'account_holder_postal_code',
                   'account_holder_city', 'account_holder_country',
-                  'account_iban', 'account_bic', 'account_number',
-                  'account_bank_name', 'account_bank_address',
-                  'account_bank_postal_code', 'account_bank_city',
-                  'account_bank_country', 'account_other', 'tasks')
+                  'account_number', 'account_bic',
+                  'account_bank_country', 'tasks')
```
Comment (eodolphi) [inline bluebottle/bb_projects/serializers.py]: Do we still need the country? Can that be different from account_holder_country?

File: bluebottle/bb_projects/models.py (hunk 2 above)
Comment (gannetson) [inline bluebottle/bb_projects/models.py]: You already did the changes to reef, right?

File: bluebottle/bb_projects/serializers.py (hunk 3 above)
Comment (gannetson) [inline bluebottle/bb_projects/serializers.py]: Yeah, country should be in as per Joris.

File: bluebottle/bb_projects/models.py (hunk 2 above)
Comment (eodolphi) [inline bluebottle/bb_projects/models.py]: Yeah, nut it does not match exactly. There it is called account_iban. Which is probably wrong.

Comment (gannetson) [inline bluebottle/bb_projects/models.py]: Yes because it can be a non-iban too

**Pull request merged**